orgs/feature.json

```diff
@@ -1,6 +1,7 @@
 {
   "orgName": "Outbound Funds (npsp) - Feature Test Org",
   "edition": "Developer",
+  "instance": "cs46",
```

Comment:
I think it's safe for now to point `feature` to prerelease pods while we wait for all pods to move to `50.0`.

Reply:
We don't need this right now because MetaCI will take care of it for you. I'd prefer to remove it because it will be spurious after tomorrow.
```suggestion
```